### DISCUSSION TITLE
Upgrade tw-element from 1.0.0-alpha12 to v1.1.0 in rails 6

### DISCUSSION THREAD

**Discussion by DhanBirSunwar** [mdbootstrap/TW-Elements#2292] [Chat about dev stuff]
I have upgrade a tw-element from 1.0.0-alpha12 to v1.1.0 in rails 6 javascript is not loading. can any one help me why is the issue?

**Comment by iprzybysz** [comment]
Hi @DhanBirSunwar, please describe your issue clearly, and add details about it - provide a snippet of your code, configuration and content of your error messages if there are any.
